README.md

```diff
@@ -77,5 +77,16 @@ The value of the option is in the [Go duration format][2].
 
 Any invalid values used for `agent.hotplug_timeout` will fall back to the default of 3 seconds.
 
+## Container Pipe Size
+
+The agent will configure a [Pipe][3] for stdio (stdout, stderr, stdin) for each container. By default,
```

Comment:
Is this being done for stdin as well? I see below that the pipe is extended just for stdout and stderr. 
I am fine with not extending stdin if it is not required.

Reply:
It's only being done for stdout/stderr. I found stdin not to be required.

Reply:
Does this line need to be updated to reflect that then?